packages/arb-token-bridge-ui/tests/e2e/specs/urlQueryParam.cy.ts

```diff
@@ -119,7 +117,7 @@ describe('User enters site with query params on URL', () => {
         cy.waitUntil(
           () =>
             cy
-              .findByPlaceholderText(/Enter amount/i)
+              .findByPlaceholderText(/0/i)
```

Comment:
Same goes with all instances of `findByPlaceholderText(0)`. We should add a utility function like `cy.getAmountInputText()` and catch it through `aria-label` instead of `placeholder = 0`.